lib/arrdep2connections.js

```diff
@@ -14,8 +13,7 @@ function ArrDep2Connections (arrivalsStream, options) {
 }
 
 ArrDep2Connections.prototype._transform = function (departure, encoding, done) {
-  departure = JSON.parse(departure.value);
-  departure["departureTime"] = new Date(departure["departureTime"]);
+  departure["gtfs:departureTime"] = departure["date"] + 'T' + departure["gtfs:departureTime"];
```

Comment:
then it's not a `gtfs:departureTime` anymore ;-) then it's a `lc:departureTime` though